src/toml/toml_lexer.zig

```diff
@@ -801,7 +805,7 @@ pub const Lexer = struct {
         var buf = buf_.*;
         defer buf_.* = buf;
 
-        const iterator = strings.CodepointIterator{ .bytes = text[start..], .i = 0 };
+        const iterator = strings.CodepointIterator{ .bytes = text, .i = 0 };
```

Comment:
We don't need `[start..]`? My only concern is I see a few places in `next` where `string_literal_slice` is set with `start + 2..`